README.md

```diff
@@ -129,10 +129,14 @@ In this case, your `application.html.erb` would contain lines like these:
 You can change the path to the manifest file with the following configuration:
 
 ```ruby
-config.x.faucet_pipeline.manifest_path = File.join("my", "own", "manifests", "path.json")
+config.faucet_pipeline.manifest_path = Rails.root.join("my", "own", "manifests", "path.json")
```

Comment:
Perhaps we should use `"public/assets/manifest.json"` as the recommended path here rather than some awkward pseudo-example (conventions, you see... )?

Reply:
But... that's the default? Why would you re-set it to the default?

Reply:
OIC, I misunderstood that bit. The example still seems contrived. 🤷‍♂️